### PR TITLE
Milestone: Database performance audit

### PR DESCRIPTION
**Pull request by geekelo**
BEFORE OPTIMIZATION:
1. 
image

2. 
image

3. 
image

AFTER OPTIMIZATION:
1. 
image

2. 
image

3. 
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -205,6 +205,12 @@ GROUP BY s.specie_name
...
-
+-- QUERIS TO CHECK PERFORMANCE
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+-- queries for optimizing the visits and owners tables
+CREATE INDEX idx_visits_animal_id ON visits(animal_id);
+CREATE INDEX idx_visits_vet_id ON visits(vet_id);
+CREATE INDEX idx_owners_email ON owners(email);
```
Comment (V-Blaze) [inline queries.sql]: - You have done great so far :+1: :100: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- [ ] You have done well using INDEX to improve the execution time of your query, but these additions of indexes should be done on the `schema.sql` file and not in the queries.sql. :+1: :+1:

File: README.md [hunk 2]
```diff
@@ -183,7 +183,10 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/geekelo_xyz)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/eloghene-otiede)
 
+👤 **Mahdi Noori**
 
+- GitHub: [@Mahdi Noori](https://github.com/mahdinoori2000)
+- LinkedIn: [@Mahdi Noori](https://www.linkedin.com/in/mahdi-noori-hc201/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (V-Blaze) [inline README.md:181]: - [ ] The MIT license link in your readme leads to a 404 error page, ill suggest you add a license file and link it correctly as broken links make readme look a bit unprofessional :heavy_check_mark: :+1: 

- - Screenshot:
Screenshot from 2023-09-05 10-54-17


- [ ] Also I'll suggest you go through your readme and remove these placeholder comments as they were added to guide you through and you were meant to remove them while customizing the readme. :smile: :small_airplane: 

- - Screenshot:
 
Selection_016

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @geekelo, @mahdinoori2000  

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Added screenshot of query results✔
-  Used INDEX to improve search time👋 
-  Correct Git flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**